### ISSUE TITLE
Training fails without any reason

### ISSUE THREAD

**Issue by GasolSun36** [OpenRLHF/OpenRLHF#545]
I am training a reward model, but after starting the training, it is killed immediately after running a step, without any explicit reason. What could be the reason? I have 8 H20 and did not use docker, but installed it myself from conda environment.

What is shown in command line:
`LogSigmoid Loss
Train epoch:   0%|                                                                                                                                                                                                                                                                | 0/1 [00:00<?, ?it/s[2024-11-29 07:06:01,711] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12476                                                            | 1/69362 [00:11<217:12:34, 11.27s/it, loss=0.693, acc=0.375, chosen_reward=-0.0443, reject_reward=-0.0442, loss_mean=0.0693, acc_mean=0.0375, lr=0]
[2024-11-29 07:06:02,208] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12477
[2024-11-29 07:06:02,784] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12478
[2024-11-29 07:06:03,280] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12479
[2024-11-29 07:06:03,737] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12480
[2024-11-29 07:06:04,233] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12481
[2024-11-29 07:06:04,728] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12482
[2024-11-29 07:06:05,385] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 12483
[2024-11-29 07:06:05,385] [ERROR] [launch.py:325:sigkill_handler] ['/usr/local/conda/bin/python', '-u', '-m', 'openrlhf.cli.train_rm', '--local_rank=7', '--save_path', './checkpoint/llama3-8b-rm', '--save_steps', '-1', '--logging_steps', '1', '--eval_steps', '-1', '--train_batch_size', '256', '--micro_train_batch_size', '1', '--pretrain', '/mnt/data/sunjiashuo/llama-sft', '--bf16', '--max_epochs', '1', '--max_len', '8192', '--zero_stage', '3', '--learning_rate', '9e-6', '--dataset', 'OpenRLHF/preference_dataset_mixture2_and_safe_pku', '--apply_chat_template', '--chosen_key', 'chosen', '--rejected_key', 'rejected', '--flash_attn', '--packing_samples', '--gradient_checkpointing'] exits with return code = -8`


my script:

`
deepspeed --module openrlhf.cli.train_rm \
   --save_path ./checkpoint/llama3-8b-rm \
   --save_steps -1 \
   --logging_steps 1 \
   --eval_steps -1 \
   --train_batch_size 256 \
   --micro_train_batch_size 1 \
   --pretrain llama-sft \
   --bf16 \
   --max_epochs 1 \
   --max_len 8192 \
   --zero_stage 3 \
   --learning_rate 9e-6 \
   --dataset OpenRLHF/preference_dataset_mixture2_and_safe_pku \
   --apply_chat_template \
   --chosen_key chosen \
   --rejected_key rejected \
   --flash_attn \
   --packing_samples \
   --gradient_checkpointing
`

**Comment by GasolSun36** [comment]
btw, sft was also killed without any reason.

**Comment by hijkzzz** [comment]
env issue

**Comment by GasolSun36** [comment]
> env issue

Is there any package I haven't installed? I installed it by git and `pip install -e .`
